projectroles/migrations/0026_delete_category_settings.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.2.18 on 2023-03-14 10:21
+
+from django.db import migrations
+from projectroles.app_settings import AppSettingAPI
+
+
+def clean_up_app_settings(apps, schema_editor):
+    app_settings_api = AppSettingAPI()
+    AppSetting = apps.get_model('projectroles', 'AppSetting')
+
+    # Find all app settings with project type 'CATEGORY'
+    app_settings = AppSetting.objects.filter(project__type='CATEGORY')
+
+    for app_setting in app_settings:
+        try:
+            setting = app_settings_api.get_definition(
+                app_name=app_setting.app_plugin.name, name=app_setting.name
+            )
+        except ValueError:
+            # AppSetting is already deleted
```

Comment:
This is not exactly correct. `get_definition()` only returns the dict from a plugin (or projectroles). It is possible for "orphaned" `AppSetting` objects to remain in the database if a definition has been removed from the plugin/projectroles dict. The `cleanappsettings` management command usually takes care of these.

Reply:
But in that case I receive such an error:
ValueError: App setting not found in app "example_project_app" with name "project_category_str_setting"
I think as ValueError can be raised in get_definition(), we have to have this exception.